dianna/dashboard/Home.py

```diff
@@ -16,26 +18,64 @@
                         'https://github.com/dianna-ai/dianna')
                    })
 
-add_sidebar_logo()
+# Define dictionary of dashboard pages
+pages = {
+    "Home": "home",
+    "Images": "pages.Images",
+    "Text": "pages.Text",
+    "Time series": "pages.Time_series"
+}
 
-st.image(str(data_directory / 'logo.png'))
+# Set up the top menu
+selected = option_menu(
+    menu_title=None,
+    options=["Home", "Images", "Text", "Time series"],
+    icons=["house", "camera", "alphabet", "clock"],
+    menu_icon="cast",
+    default_index=0,
+    orientation="horizontal"
+)
 
-st.markdown("""
-DIANNA is a Python package that brings explainable AI (XAI) to your research project.
-It wraps carefully selected XAI methods in a simple, uniform interface. It's built by,
-with and for (academic) researchers and research software engineers working on machine
-learning projects.
+# Display the content of the selected page
+if selected == "Home":
+    add_sidebar_logo()
 
-### Pages
+    st.image(str(data_directory / 'logo.png'))
 
-- <a href="/Images" target="_parent">Images</a>
-- <a href="/Text" target="_parent">Text</a>
-- <a href="/Time_series" target="_parent">Time series</a>
+    st.markdown("""
+    DIANNA is a Python package that brings explainable AI (XAI) to your research project.
+    It wraps carefully selected XAI methods in a simple, uniform interface. It's built by,
+    with and for (academic) researchers and research software engineers working on machine
+    learning projects.
 
+    ### Pages
 
-### More information
+    - <a href="/Images" target="_parent">Images</a>
+    - <a href="/Text" target="_parent">Text</a>
+    - <a href="/Time_series" target="_parent">Time series</a>
```

Comment:
Do we still need this list? I know removing this isn't a part of issue #817, but perhaps we can also remove them here.

Reply:
Fair point, I think it is best to address this in [#816](https://github.com/dianna-ai/dianna/issues/816)